easybuild/easyconfigs/m/mkl-dnn/mkl-dnn-0.16-foss-2018b.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'mkl-dnn'
+version = '0.16'
+
+homepage = 'https://01.org/mkl-dnn'
+description = "Intel(R) Math Kernel Library for Deep Neural Networks (Intel(R) MKL-DNN)"
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [
+    'https://github.com/01org/mkl-dnn/archive/',
+    'https://github.com/intel/mkl-dnn/releases/download/v%(version)s',
+]
+mklml_extract_cmd = "mkdir %(builddir)s/mkl-dnn-%(version)s/external/ && "
+mklml_extract_cmd += "tar xvfz %s -C %(builddir)s/mkl-dnn-%(version)s/external/"
+sources = [
+    'v%(version)s.tar.gz',
+    # mkl-dnn requires Intel MKL, so download minimal Intel MKL version
+    # actually using imkl as a dependency would be messy due to underlying icc/ifort compilers & impi MPI lib
```

Comment:
what about using the gomkl toolchain?

Reply:
That's another option, but I don't see the added benefit?

Reply:
well, in terms of using it mkl as a dependency, it would be clearer... 

but given that the easyconfig itself is for mkl(-dnn) and that I suppose nobody will want/try to use mkl-dnn with, say, openblas, I suppose it's ok